python_modules/dagster/dagster/_core/storage/runs/sql_run_storage.py

```diff
@@ -189,19 +191,17 @@ def handle_run_event(self, run_id: str, event: DagsterEvent) -> None:
 
         kwargs = {}
 
-        # consider changing the `handle_run_event` signature to get timestamp off of the
-        # EventLogEntry instead of the DagsterEvent, for consistency
-        now = get_current_datetime()
+        update_timestamp = update_timestamp or get_current_datetime()
```

Comment:
comments here or below to clarify what "update_timestamp" means to us ie not when the db row was last touched but the time of the latest event to change the status of the run